core/src/Alert.html

```diff
@@ -0,0 +1,99 @@
+<div
+  class="fd-alert fd-alert--{settings.type} {settings.dismissButton ? 'fd-alert--dismissible' : ''}"
+  role="alert"
```

Comment:
fd-alert--info does not work. It should be fd-alert--information as far as I can see.

Reply:
done